src/Tests/dotnet-new.Tests/Approvals/DotnetNewHelp.CannotShowHelpForTemplate_PartialNameMatch.verified.txt

```diff
@@ -1,4 +1,6 @@
-﻿No templates found matching: 'class'.
+﻿No templates or subcommands found matching: 'class'. Did you mean one of the following?
+   classlib
+   list
```

Comment:
I think that having the subcommands and templates in a combined list like this is going to be pretty confusing.

How hard is it to introduce some separation between the templates and subcommands?

```
No templates or commands found matching: 'class'. Did you mean one of the following templates?
     classlib
Or one of the following commands?
     list
```

Reply:
This won't be hard. btw do you think it would be better to include the whole command:
```
No templates or commands found matching: 'class'. Did you mean to run one of the following templates?
     dotnet new classlib
Or one of the following commands?
     dotnet new list
```

Slightly more words, but easier to copy. We can also include other args if any.

Reply:
I like your proposal a lot for short lists for sure. I wonder what it would look like for longer lists? Here is 6 matching templates:

```
No templates or commands found matching: 'class'. Did you mean to run one of the following templates?
     dotnet new classlib1
     dotnet new classlib2
     dotnet new classlib3
     dotnet new classlibs
     dotnet new classlib4
     dotnet new classlib5
Or one of the following commands?
     dotnet new list
```

That looks better than I was expecting - I was thinking that the repeated 'dotnet new' would be a lot of visual noise.  Let's go with the full commands for now.

Reply:
+1 for usability of full commands read-made for copy-paste-run usage

Reply:
Pushed the update:
- for regular command the experience is:
```
> dotnet new class or > dotnet new class --framework net7.0
﻿No templates or subcommands found matching: 'class'.
Did you mean one of the following templates?
   dotnet new classlib
Did you mean one of the following subcommands?
   dotnet new list
```

- for command with help flag the experience is:
```
> dotnet new class -h
﻿No templates or subcommands found matching: 'class'.
Did you mean one of the following templates?
   dotnet new classlib -h
Did you mean one of the following subcommands?
   dotnet new list -h
```

I preferred not to add other options to examples, as they may be incorrect for suggested command.  The only special case is help. imho, with having complete command having two lists might be too much of noise, but looking for the feedback.